src/dao.ts

```diff
@@ -383,7 +382,7 @@ export class Dao {
 export class DaoMember {
   constructor(
     public dao: Dao,
-    private client: SigningCosmWasmClient,
+    private client: SigningCosmWasmClient | Eip191SigningCosmwasmClient,
```

Comment:
why is it `SigningCosmWasmClient | Eip191SigningCosmwasmClient`? I thought you wanted to remove `SigningCosmWasmClient` usage from our code at all and replace it totally with the universal client